Podfile

```diff
@@ -0,0 +1,52 @@
+platform :ios, '10.0'
+use_frameworks!
+
+def testing_pods
+  pod 'RxSwift'
+  pod 'Quick'
+  pod 'Nimble'
+  pod 'RxNimble', subspecs: ['RxBlocking', 'RxTest']
+  pod 'Sourcery'
+end
+
+target '{PROJECT_NAME}' do
+  # UI
+  pod 'SnapKit'
+  pod 'Kingfisher'
+
+  # Rx
+  pod 'RxSwift'
+  pod 'RxCocoa'
+  pod 'RxDataSources'
+  pod 'RxAlamofire'
```

Comment:
I think we can sort these pods alphabetically for each section as well? What do you think? 😁